build.gradle

```diff
@@ -378,16 +407,18 @@ subprojects {
     plugins.apply('idea')
     plugins.apply('eclipse')
     plugins.apply('maven')
-    plugins.apply('findbugs')
 
     sourceCompatibility = JavaVersion.VERSION_1_7
 
-    findbugs {
-      toolVersion = findBugsVersion
-      ignoreFailures = true
-      effort = "max"
-      // The exclude filter file must be under "ligradle/findbugs/"
-      excludeFilter = file(rootProject.projectDir.path + "/ligradle/findbugs/findbugsExclude.xml")
+    if (project.hasProperty("disableFindBugs")) {
```

Comment:
Shouldn't it be `if (!project.hasProperty("disableFindBugs"))` - also is this necessary? You can always disable findBugs by running `./gradlew assemble` instead of `./gradlew build`